DATA/common/setenv_calib.sh

```diff
@@ -11,11 +11,30 @@
 if [[ -z $SOURCE_GUARD_SETENV_CALIB ]]; then
 SOURCE_GUARD_SETENV_CALIB=1
 
+# by default, all calibrations are disabled, because we don't know which detectors are in the data taking
+[[ -z ${CALIB_PRIMVTX_MEANVTX} ]] && CALIB_PRIMVTX_MEANVTX=0
+[[ -z ${CALIB_TOF_LHCPHASE} ]] && CALIB_TOF_LHCPHASE=0
+[[ -z ${CALIB_TOF_CHANNELOFFSETS} ]] && CALIB_TOF_CHANNELOFFSETS=0
+[[ -z ${CALIB_TOF_DIAGNOSTICS} ]] && CALIB_TOF_DIAGNOSTICS=0
+[[ -z ${CALIB_TPC_SCDCALIB} ]] && CALIB_TPC_SCDCALIB=0
+[[ -z ${CALIB_TPC_TIMEGAIN} ]] && CALIB_TPC_TIMEGAIN=0
+[[ -z ${CALIB_TPC_RESPADGAIN} ]] && CALIB_TPC_RESPADGAIN=0
+[[ -z ${CALIB_TPC_IDC} ]] && CALIB_TPC_IDC=0
```

Comment:
you removed the force-disable here, in case TPC is not present.
But this is needed.
Otherwise the current cosmics, which have CALIB_TPC_IDC=1 hardcoded will fail, since they run out of calib nodes.
I think, in this default setting, we should check for all detectors whether they are present, and if not override it to =0 even if it is already set to =1 (like I had done below for TPC, what you removed now)